appveyor.yml

```diff
@@ -1,33 +1,45 @@
-build: false
+# https://ci.appveyor.com/project/goanpeca/pyswmm
+
+matrix:
+  fast_finish: true
+
+branches:
+  only:
+    - master 
 
 environment:
+  global:
+    # Used by qthelpers to close widgets after a defined time
+    TEST_CI: True
+    # Environment variables used by astropy helpers
+    PYTHON: "C:\\conda"
+    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci-helpers\\appveyor\\windows_sdk.cmd"
+    PYTHON_ARCH: "64" # needs to be set for CMD_IN_ENV to succeed. If a mix
```

Comment:
@goanpeca, maybe I'm missing something.  As i understand, a 64-bit library cannot talk to a 32-bit.  So if you are using 64-bit python, it cannot talk talk to the 32-bit SWMM dll. I've been wrong many times in my life so let me know if that is the case now ;)